.github/workflows/reusable-mac-ci.yml

```diff
@@ -85,35 +85,21 @@ jobs:
       - name: Prepare colcon workspace
         id: colcon_ws_setup
         run: |
-          # Nightly job
-          if [[ "${{ inputs.label }}" == *"nightly"* ]]
-          then
-            DEPENDS_REPOS_PATH="./src/fastrtps/.github/workflows/config/nightly_${{ inputs.fastdds-branch }}.repos"
-            if [ ! -f ${DEPENDS_REPOS_PATH} ]
-            then
-              DEPENDS_REPOS_PATH="./src/fastrtps/.github/workflows/config/nightly_master.repos"
-            fi
-            META_PATH="./src/fastrtps/.github/workflows/config/nightly.meta"
-
-          # Either PR or manual
-          else
-            DEPENDS_REPOS_PATH="./src/fastrtps/.github/workflows/config/default_ci_${{ inputs.fastdds-branch }}.repos"
-            if [ ! -f ${DEPENDS_REPOS_PATH} ]
-            then
-              DEPENDS_REPOS_PATH="./src/fastrtps/.github/workflows/config/default_ci_master.repos"
-            fi
-            META_PATH="./src/fastrtps/.github/workflows/config/default_ci.meta"
-          fi
-
-          echo "Selected repos files: ${DEPENDS_REPOS_PATH}"
-          cat ${DEPENDS_REPOS_PATH}
-
-          echo "Selected metas files: ${META_PATH}"
+          REPOS_PATH="./src/fastrtps/.github/workflows/config/ci.repos"
+          META_PATH="./src/fastrtps/.github/workflows/config/ci_colcon.meta"
+
+          echo "CI repos file:"
+          cat ${REPOS_PATH}
+
+          echo "CI metas file:"
           cp ${META_PATH} ci.meta
           cat ci.meta
 
-          # Create source dir and download the sources
-          vcs import src --input ${DEPENDS_REPOS_PATH} --skip-existing
+          # Create source dir and download the version sources
+          vcs import src --input ./src/fastrtps/fastrtps.repos --skip-existing
+
+          # Download the CI sources
+          vcs import src --input ${REPOS_PATH} --skip-existing
```

Comment:
I'd transform this into 2 calls to eprosima-CI action. Mind that you'dd need to adjust the path to the meta file on the build step. Same for all workflows